README.md

```diff
@@ -25,6 +25,15 @@ You can make and test client changes simply by opening `testclient.html`.
 This will allow you to test changes to the client without setting up your
 own login server.
 
+However, you will have to retrieve the latest data files first, which are stored
+only in the [Pokémon Showdown server][3] repository.
+
+```bash
+git clone https://github.com/Zarel/Pokemon-Showdown.git data/Pokemon-Showdown --depth=1
+cd githooks
+node build-indexes
```

Comment:
The intended use is `node githooks/build-indexes` which should probably be renamed to something like `build/build` anyway